test/build_graph.cpp

```diff
@@ -25,6 +25,16 @@
 using namespace std;
 using namespace ngraph;
 
+TEST(build_graph, broadcast_to_non_existent_axis)
```

Comment:
FWIW this test can probably be dropped altogether. It seems to be a less precise version of `type_prop.broadcast_axes_oob`.